index.html

```diff
@@ -167,86 +175,132 @@ <h2>Verifiable Credential</h2>
 
     <section>
       <h2>Securing JSON-LD</h2>
+      <p>
+        The use of Verifiable Credentials often involves the representation and exchange of structured 
+        data in the form of JSON-LD. While JSON-LD provides a flexible and extensible format for 
+        describing data, it is also important to note that it provides a linkage between the data structure
+        and semantic meaning of data.
+        This section outlines how JSON-LD encoded data can be secured using either JOSE or COSE though use of
+        a VC-JWT. 
+      </p>
+      <p>
+        A benefit to this approach is that credentials can be used that conform directly to the [[VC-DATA-MODEL]]
+        without need of any mapping or transformation.
+      </p>
       <section>
         <h2>With JOSE</h2>
         <section>
-          <h2>Securing <code>application/vc+ld+json</code> with JOSE</h2>
+          <h2>Securing JSON-LD VCs with JOSE</h2>
+          <p>
+            This section details how to secure data with the type <code>application/vc+ld+json</code> 
```

Comment:
```suggestion
            This section details how to secure data documents with the type <code>application/vc+ld+json</code> 
```

Reply:
"data documents" is not really the right term, we are securing `claimset` or `payload`, which is a JSON-LD document.

Reply:
We should not use `claimset` in the case the payload is `application/vc+ld+json` or it could be confused with something where the IANA JWT claims registry could be erroneously used to try and understand the JSON-LD document. I think `payload` works though.